_deploy/r/gnoswap/common/tick_math.gno

```diff
@@ -43,34 +43,40 @@ var binaryLogConsts = [8]*u256.Uint{
 
 var (
 	shift1By32Left = u256.MustFromDecimal("4294967296") // (1 << 32)
+	maxTick        = int32(887272)
 )
 
 func TickMathGetSqrtRatioAtTick(tick int32) *u256.Uint { // uint160 sqrtPriceX96
 	absTick := abs(tick)
-	if absTick > 887272 { // MAX_TICK
+	if absTick > maxTick { // MAX_TICK
 		panic(addDetailToError(
 			errOutOfRange,
 			ufmt.Sprintf("tick_math.gno__TickMathGetSqrtRatioAtTick() || tick is out of range (larger than 887272), tick: %d", tick),
 		))
 	}
 
-	ratio := u256.MustFromDecimal("340282366920938463463374607431768211456") // consts.Q128
+	var initialBit int32 = 0x1
+	var ratio *u256.Uint
```

Comment:
```suggestion
	ratio := u256.Zero()
```

I think when using new u256 variable, it's better to init with 0 value to avoid nil pointer error just in case.

Reply:
@r3v4s This is true when calling internal functions via ratio, but the following logic should never cause a nil pointer error. In this case, we don't need to do any unnecessary initialization.